--- GNUMail/Bundles/Filtering/FilterEditorWindowController.m.orig	2003-12-25 17:00:25.000000000 +0100
+++ GNUMail/Bundles/Filtering/FilterEditorWindowController.m	2004-07-04 22:39:07.565766664 +0200
@@ -38,7 +38,7 @@
 #include <Pantomime/LocalStore.h>
 #include <Pantomime/NSString+Extensions.h>
 
-#include <AddressBook/AddressBook.h>
+#include <Addresses/AddressBook.h>
 
 
 #ifndef MACOSX
--- GNUMail/AddressBookController.h.orig	2003-11-23 01:25:42.000000000 +0100
+++ GNUMail/AddressBookController.h	2004-07-04 22:40:05.898772062 +0200
@@ -24,7 +24,7 @@
 #define _GNUMail_H_AddressBookController
 
 #import <AppKit/AppKit.h>
-#import <AddressBook/AddressBook.h>
+#import <Addresses/AddressBook.h>
 
 #ifndef MACOSX
 @class ADSinglePropertyView;
